physical/s3/s3.go

```diff
@@ -52,6 +53,11 @@ func NewS3Backend(conf map[string]string, logger log.Logger) (physical.Backend,
 		}
 	}
 
+	path, ok := conf["path"]
```

Comment:
Can be simplified to `path := conf["path"]`

Reply:
'tis true